packages/frontend/utils/apollo.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable */
+import React from 'react';
+import Head from 'next/head';
+import { ApolloProvider } from '@apollo/react-hooks';
+import { ApolloClient } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
+import fetch from 'isomorphic-unfetch';
+
+let apolloClient = null;
+
+/**
+ * Creates and provides the apolloContext
+ * to a next.js PageTree. Use it by wrapping
+ * your PageComponent via HOC pattern.
+ * @param {Function|Class} PageComponent
+ * @param {Object} [config]
+ * @param {Boolean} [config.ssr=true]
+ */
+export function withApollo(PageComponent, { ssr = true } = {}) {
```

Comment:
Looking forward to seeing the tests for this 🕺

Reply:
it is hard but i will try 😄

Reply:
@msusur i added some tests but we are on lower coverage. for apollo, there is 50% statements, 60% branches, 80% functions and 50% lines covered.